internal/config/projectconfig/project_config.go

```diff
@@ -29,6 +30,24 @@ type Module struct {
 	Conditions []Condition `yaml:"conditions,omitempty"`
 }
 
+// ReadVendorCredentialsFromModule uses parsed project-config's module
+// based on vendor parameter, retrieve the vendor's credential
+// for pre-defined functionalities (eg: Github api key for pushing repos to github)
+func ReadVendorCredentialsFromModule(m Module, vendor string) (error, string) {
+	// this mapping could be useful for module config as well
+	vendorToParamMap := map[string]string{
+		"github":   "githubAccessToken",
```

Comment:
Where are these vendor strings going to come from?

Reply:
the function is actually only called for vcs (during `zero create`) right now, I didnt want to have the logic there on how to get the `github-api-key` from project config. 
so the `circleci` part is actually never used for now.